services/movie.py

```diff
@@ -0,0 +1,44 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> tuple[Movie]:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+    if genres_ids and not actors_ids:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids
+        )
+    if not genres_ids and actors_ids:
+        return Movie.objects.filter(
+            actors__id__in=actors_ids
+        )
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            actors__id__in=actors_ids,
+            genres__id__in=genres_ids
+        )
+
+
+def get_movie_by_id(
+        movie_id: int
+) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list = None,
+        actors_ids: list = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]
+
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]